packages/cluster/src/common.js

```diff
@@ -88,9 +100,28 @@ function resolveConfiguration(argv = {}, cache = {}, config = {}) {
 
     reporter.debug('Resolved configuration\n', resolved)
 
+    await argv.getCache().write(
```

Comment:
Same deal here, the `Cache.write` just logs an error and continues if the cache write fails. I don't think that it's such a big deal if the cache fails to be written, and I want the behaviours of the `read`/`write` pair to match, so if one throws, the other should throw. Since the implementation of `read` returns `null` if it fails, I let this one log & continue on a cache write fail.